### ISSUE TITLE
Crash after selection of information icon on toolbar

### ISSUE THREAD

**Issue by sladepb** [qgis/QGIS#58326]
## User Feedback

A red cross was present from a prevcious action. I then selected the the "i" information icon and QGIS crashed...

## Report Details

**Python Stack Trace**
```
Windows fatal exception: access violation

Current thread 0x00005dfc (most recent call first):
  <no Python frame>
```

**Stack Trace**

```

QObjectPrivate::addConnection :
QObjectPrivate::connectImpl :
QObject::connectImpl :
QgsVertexTool::cachedGeometry :
QgsVertexTool::moveVertex :
QgsVertexTool::cadCanvasReleaseEvent :
QgsMapToolAdvancedDigitizing::canvasReleaseEvent :
QgsMapCanvas::mouseReleaseEvent :
QWidget::event :
QFrame::event :
QGraphicsView::viewportEvent :
QCoreApplicationPrivate::sendThroughObjectEventFilters :
QApplicationPrivate::notify_helper :
QApplication::notify :
QgsApplication::notify :
QCoreApplication::notifyInternal2 :
QApplicationPrivate::sendMouseEvent :
QSizePolicy::QSizePolicy :
QSizePolicy::QSizePolicy :
QApplicationPrivate::notify_helper :
QApplication::notify :
QgsApplication::notify :
QCoreApplication::notifyInternal2 :
QGuiApplicationPrivate::processMouseEvent :
QWindowSystemInterface::sendWindowSystemEvents :
QEventDispatcherWin32::processEvents :
qt_plugin_query_metadata :
QEventLoop::exec :
QCoreApplication::exec :
main :
BaseThreadInitThunk :
RtlUserThreadStart :
```

**QGIS Info**
QGIS Version: 3.36.3-Maidenhead
QGIS code revision: 2df96554
Compiled against Qt: 5.15.13
Running against Qt: 5.15.13
Compiled against GDAL: 3.9.0
Running against GDAL: 3.9.0

**System Info**
CPU Type: x86_64
Kernel Type: winnt
Kernel Version: 10.0.26120

### Steps to reproduce the issue

A red cross was present from a prevcious action. I then selected the the "i" information icon and QGIS crashed...

### Versions

Refer paste from report above

### Supported QGIS version

- [x] I'm running a supported QGIS version according to [the roadmap](https://www.qgis.org/en/site/getinvolved/development/roadmap.html#release-schedule).

### New profile

- [ ] I tried with a new [QGIS profile](https://docs.qgis.org/latest/en/docs/user_manual/introduction/qgis_configuration.html#working-with-user-profiles)

### Additional context

A red cross was present from a prevcious action. I then selected the the "i" information icon and QGIS crashed...

**Comment by nicogodet** [comment]
3.36 is not supported anymore, could you update your QGIS and see if it still happens ?

If so, provide a sample project to reproduce the crash please.

**Comment by sladepb** [comment]
The problem did not recur with 3.38. This is now closed.

**Comment by sladepb** [comment]
Closed as problem can not be reproduced on 3.38
